Frontend/src/Components/Graph/ExpandableContent.js

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import { Collapse, Box, Button } from "@chakra-ui/react";
+import { ChevronDownIcon } from "@chakra-ui/icons";
+
+function Expandable(props) {
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  return (
+    <Box maxW="2xl">
+      <Box>
+        <Button
+          onClick={() => setIsExpanded(!isExpanded)}
+          leftIcon={<ChevronDownIcon />}
+          variant="ghost"
+          colorScheme="gray"
+          size={props.size}
+        >
+          {props.label}
+        </Button>
+      </Box>
```

Comment:
```suggestion
      <Button
        onClick={() => setIsExpanded(!isExpanded)}
        leftIcon={<ChevronDownIcon />}
        variant="ghost"
        colorScheme="gray"
        size={props.size}
        w="100%"
        justifyContent="left"
      >
        {props.label}
      </Button>
```
A few things:
1. The outer Box is unnecessary
2. It would be nice if the buttons to trigger expanding the sections span the whole/most of the width of the modal.
   1. Adding the `w` and `justifyContent` props could do this.

Reply:
fixed